api/app/Http/Controllers/AuthController.php

```diff
@@ -70,7 +72,10 @@ public function authCallback(Request $request)
         // pull token out of the response as json -> lcobucci parser, no key verification is being done here however
         $idToken = $response->json("id_token");
 
-        $config = Configuration::forUnsecuredSigner();
+        $config = Configuration::forSymmetricSigner(
+            new Signer\Blake2b(),
+            InMemory::base64Encoded('MpQd6dDPiqnzFSWmpUfLy4+Rdls90Ca4C8e0QD0IxqY=')
```

Comment:
I don't quite get why we want to add it in when the dependency person seemed adamant that None is bad practice

Reply:
I think this library was written for implementing your own auth provider.  In our case we just want to decode the token to get some data out of it.  Since we didn't actually issue the token (it was issued by GCKey) we don't have the private key to verify the signing.    But since we're not the auth provider it doesn't really matter to us.

Unfortunately, it seem like this library no longer has a clean way to decode a token without using a private key.  Maybe, someday, we'll switch to a frontend auth library.  :laughing: Then we won't have to deal with the weird corner cases of our "sidecar" implementation.

Reply:
😖

Reply:
Yeah, the injected service with the function looks great! :1st_place_medal: